tests/cli/test_cmds_check.py

```diff
@@ -285,6 +287,51 @@ def test_check_smartstack_check_is_ok_when_no_smartstack(mock_is_file_in_dir, ca
     assert output == expected_output
 
 
+@patch("paasta_tools.cli.cmds.validate.get_config_file_dict", autospec=True)
+def test_check_parallel_works(mock_get_config_file_dict):
+    # Parallel steps is seen as valid
+    mock_get_config_file_dict.return_value = {
+        'pipeline': [{
+            'parallel': [
+                {
+                    'step': 'somecluster.something',
+                },
+                {
+                    'step': 'somesecurity.check',
+                },
+            ],
+            'step': 'anothercluster.something',
+            'wait_for_deployment': False,
+            'timeout': 3600,
+        }],
+    }
+
+    actual = validate_schema('my/fake-service','deploy') is True
+    assert actual is True
+
+@patch("paasta_tools.cli.cmds.validate.get_config_file_dict", autospec=True)
+def test_check_parallel_fails(mock_get_config_file_dict):
+    # Parallel steps is seen as valid
```

Comment:
if we don't go for the suggestion below, this comment is inaccurate (looks like it wasn't changed when basing it off the test above?)